### PR TITLE
Postgres ops

### PR DESCRIPTION
**Pull request by rossf7**
Pawel,
here is the postgresops library and the Postgres deployment. There is also a basic main.go that shows it being used.

I'm AFK tomorrow so feel free to merge this PR or just grab the postgresops code if you need it.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+}
+
+func main() {
+	err := mainWithError()
+	if err != nil {
+		log.Fatalf("error: %#v", err)
+	}
+}
+
+func mainWithError() error {
```
Comment (rossf7) [inline main.go:40]: This is a basic main that calls the methods in the postgresops package.

File: main.go [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+	if err != nil {
+		return "", fmt.Errorf("getting hostname: %s", err)
+	}
+
+	minikubeIP := strings.TrimSpace(string(out))
+
+	return minikubeIP, nil
+}
+
+func getServicePort(serviceName string) (int, error) {
```
Comment (rossf7) [inline main.go:108]: I'm using a node port rather than a cluster IP so we can run it in Go. This function just gets the port from minikube.

Comment (kopiczko) [reply main.go:108]: I didn't thought about it. This adds some complexity to remote setup, be we can deal with that.

**Review by kopiczko — APPROVED** [review]
This is cool, thanks. The only nitpick is that the name of the DB is PostgreSQL not Postgres I think.

**Comment by rossf7** [comment]
Yes the naming is confusing as sometimes Postgres is used. But I've changed to PostgreSQL as that is the correct name! :)

**Pull request merged**